.github/workflows/tests.yaml

```
@@ -30,7 +30,7 @@ jobs:
         uses: charmed-kubernetes/actions-operator@main
         with:
           provider: microk8s
-          channel: 1.25-strict/stable
+          channel: 1.28-strict/stable
```

```suggestion
          channel: 1.{{ matrix.version }}-strict/stable
```

just wondering if @natalian98 wanted to only bump or test on different releases

If that's the case adding also the below on the job config will make it happen

```
strategy:
      matrix:
        version: [25,26,27,28]
```

---

Maybe we could test individual charms just on 1.28 and the bundle on 1.27 and 1.28? I'm worried that if we test on 4 versions, the CI is going to take too long.

---

I agree, I think the integration test take too long already.

---

they would run in parallel, but the bundle tests thing sounds a good compromise